assets/dev/scss/frontend/widgets/playing-cards.scss

```diff
@@ -0,0 +1,211 @@
+		user-select: none;
+	}
+
+	.e-playing-cards-item {
+		transition: var(--e-cards-transition-time);
+		width: var(--e-card-width);
+		height: calc(var(--e-card-width) / 0.7);
+		border-radius: var(--e-card-border-radius);
+		box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 12px;
+		background-color: var(--e-card-background-color);
+		transform-style: preserve-3d;
+		cursor: pointer;
+		border: var(--e-card-border);
+
+		.e-card-backside {
+			position: relative;
+			position: absolute;
+			width: 100%;
+			height: 100%;
+			border-radius: inherit;
+			background-color: inherit;
+			transform: rotateY(.5turn);
+			padding: 10px;
+			overflow: hidden;
+
+			.image {
+				background-image: url(../images/playing-cards/backside.jpg);
+				background-size: cover;
+				width: 100%;
+				height: 100%;
+			}
+		}
+
+		&.flipped {
+			transform: rotateY(.5turn);
+
+			.e-card-backside {
+				transform: rotateY(1turn);
+			}
+		}
+
+
+		&.heart {
+			.suit {
+				color: var(--e-card-heart-color);
+			}
+
+			.value {
+				color: var(--e-card-heart-color);
+			}
+		}
+
+		&.club {
+			.suit {
+				color: var(--e-card-club-color);
+			}
+
+			.value {
+				color: var(--e-card-club-color);
+			}
+		}
+
+		&.diamond {
+			.suit {
+				color: var(--e-card-diamond-color);
+			}
+
+			.value {
+				color: var(--e-card-diamond-color);
+			}
+		}
+
+		&.spade {
+			.suit {
+				color: var(--e-card-spade-color);
+			}
+
+			.value {
+				color: var(--e-card-spade-color);
+			}
+		}
+
+		.e-card-frontside {
+			backface-visibility: hidden;
+			position: relative;
+			display: grid;
+			grid-template-columns: 1.2fr repeat( 5, 1fr ) 1.2fr;
+			grid-template-rows: 1.2fr repeat( 7, 1fr ) 1.2fr;
+			justify-items: center;
+			align-items: center;
+			width: 100%;
+			height: 100%;
+			border-radius: inherit;
+			background-color: inherit;
+			padding: var(--e-card-padding);
+			overflow: hidden;
+
+			.e-card-content {
+				font-family: var(--e-card-font-family) Sans-serif;
+				font-size: var(--e-card-font-size);
+				font-weight: var(--e-card-font-weight);
+			}
+
+			.flip {
+				rotate: .5turn;
+			}
+
+			.self-start {
+				align-self: start;
+			}
+
+			.self-end {
+				align-self: end;
+			}
+
+			.suit {
+				font-size: var(--e-card-suit-font-size);
+				line-height: 1;
+			}
+
+			.e-card-content.value {
+				font-size: var(--e-card-font-size);
+			}
+
+			.e-card-content-royal-image {
+				grid-row: 2/-2;
+				grid-column: 2/-2;
+				border: var(--e-card-border);
+				width: 100%;
+				height: 100%;
+				background-size: cover;
+				background-position: center;
+			}
+
+			.suit {
+				position: relative;
+
+				&.end {
+					align-self: end;
+					justify-self: end;
+					left: var(--e-card-royal-suit-offset-h);
+					top: var(--e-card-royal-suit-offset-v);
+				}
+
+				&.start {
+					align-self: start;
+					justify-self: start;
+					right: var(--e-card-royal-suit-offset-h);
+					bottom: var(--e-card-royal-suit-offset-v);
+				}
+			}
+
+			@mixin grid_loop() {
```

Comment:
I would think of ways to avoid the use of so many class names. We try to reduce the amount of CSS that we generate.

Reply:
I would fight the creation of so many class names in a real project.
I am curious, what do you need the class names for?
Would it not be possible to create a class something like, `.myClass-start`.